cms/src/test/resources/test-data/api-specifications/SchemaHelperTest/schemaObjectsMultiLevelHierarchy-items.json

```diff
@@ -0,0 +1,110 @@
+{
+    "openapi": "3.0.0",
+    "paths": {
+        "/test": {
+            "post": {
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+
+                            "schema": {
+                                "type": "object",
+                                "title": "object 1 - top-level schema",
+
+                                "properties": {
+                                    "property-1.1.1": {
+                                        "type": "array",
+
+                                        "items": {
+                                            "type": "object",
+                                            "title": "object 1.1.1.1 - items",
+                                            "deprecated": true,
+                                            "description": "Test Schema Object description in `Markdown`.",
+                                            "multipleOf": 1.2,
+                                            "minimum": -2.3,
+                                            "exclusiveMinimum": false,
+                                            "maximum": 2.4,
+                                            "exclusiveMaximum": true,
+                                            "minLength": 11,
+                                            "maxLength": 22,
+                                            "pattern": "^(?:[2])?",
```

Comment:
Is the non-capture group needed? Wouldn't `^2?` do the same job?

Reply:
The same situation as described in the previous comment - the actual value of the regex does not matter because all we do is display it rather than use it to match anything.